packages/doenetml-worker-javascript/vite.config.ts

```diff
@@ -1,32 +1,20 @@
 /// <reference types="vitest/config" />
-import nodePolyfills from "rollup-plugin-polyfill-node";
+import dts from "vite-plugin-dts";
 import { defineConfig } from "vite";
 import { suppressLogPlugin } from "../../scripts/vite-plugins";
 
 // https://vitejs.dev/config/
 export default defineConfig({
     base: "./",
-    plugins: [suppressLogPlugin()],
+    plugins: [suppressLogPlugin(), dts({ rollupTypes: true })],
     build: {
-        minify: true,
+        minify: false,
         sourcemap: true,
         lib: {
-            entry: "src/index.ts",
-            fileName: (_format, _entryName) => "CoreWorker.js",
-            formats: ["iife"],
-            name: "doenetmlWorker",
-        },
-        rollupOptions: {
-            // @ts-ignore
-            plugins: [nodePolyfills()],
-            external: ["react", "react-dom", "styled-components"],
-            output: {
-                globals: {
-                    react: "react",
-                    "react-dom": "react-dom",
-                    "styled-components": "styled-components",
-                },
```

Comment:
It's great we were able to get rid of these! Is this because Core and the UICore are separate now?

Reply:
Changing `minify` to `false` is causing a pretty big increase in build size, no?

Reply:
Core should never have had those dependencies. It doesn't do anything with react or the DOM. I think we just missed that when breaking up the packages before.

I switched minify to `false` as the build artifact of this package is no longer used directly. Instead, we now only use the build of `doenetml-worker` which includes `doenetml-worker-javascript`. If I remember right, we don't minify for packages that are intended to be imported by others. I assume it doesn't affect final build size as it will get minified by `doenetml-worker`. (Oops, it looks like `doenetml-worker` didn't have minify to `true`. Turning minify on `doenetml-worker` shrunk `index.js` from 16MB down to 12 MB.)